### DISCUSSION TITLE
button vs buttons

### DISCUSSION THREAD

**Discussion by vascolp** [pybricks/support#1253] [Q&A]
At last I got a Prime Hub for a good price.
I took a big program, developed on Technic/City Hubs, with all ports usage, remote control, storage usage, bluetooth talk, etc, put it in the PrimeHub and everything worked! Fantastic!
Everything?? No!
In all hubs you have the method `hub.button` but in Prime/Inventor you have `hub.buttons`.
I think it should be the same method in all hubs...
I prefer the name `buttons`, it returns a collection of buttons (even if only one).
You could probably have both names in all hubs for backward compatibility.
What do you think?

**Comment by laurensvalk** [comment]
I'd like that too. 

For backwards compatibility, we can also keep the `button` singular as an alias pointing at the same thing as `buttons`.
